### ISSUE TITLE
RSpec/PendingWithoutReason False Positive

### ISSUE THREAD

**Issue by jaredmoody** [rubocop/rubocop-rspec#1538]
If a variable named `skip` is used in a block that is not a test, a violation occurs even though the test is not skipped.

E.g.:

```ruby
describe "skip" do
  subject { skip } # violation on this line, but test runs

  let(:skip) { true }

  it { is_expected.to be(true) }
end
```

I'll grant that `skip` is probably a bad variable name, so I've renamed it in my code, but reporting it in case you want to address.

**Comment by tejasbubane** [comment]
Not sure if rubocop can handle this completely because the variable [can also be used inside example](https://relishapp.com/rspec/rspec-core/docs/pending-and-skipped-examples/skip-examples#skipping-using-%60skip%60-inside-an-example) and we don't know for sure if this is a variable or method call.

**Comment by pirj** [comment]
I think this has been fixed in #1515 and #1516, but not yet released.
